src/Navbar.tsx

```diff
@@ -128,9 +128,16 @@ class Navbar extends React.Component<any, State> {
       !hideLogin
     const isHomeSelectedCssClassName =
       window.location.pathname.replace('/', '') === '' ? 'isSelected' : ''
-    const homeRouteConfig: GenericRoute = routesConfig.filter((r) => {
-      return r.to === ''
-    })[0]
+    const homeRouteConfig: ConfigRoute | undefined = routesConfig.filter(
+      (r: GenericRoute): r is ConfigRoute => {
+        return !!(
+          r.path === '' &&
+          r.synapseConfigArray &&
+          r.synapseConfigArray.length > 0
+        )
+      },
+    )[0]
```

Comment:
Small bugfix: this renders buttons in the dropdown that link to sections of the home page. Check the length of the synapseConfigArray to see if there are actually any sections. Otherwise, the Home button will act as a dropdown with no menu items. For example, see STOP-AD portal.